src/config/middlewares/authentication.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+const jwtSecret = process.env.JWT_SECRET || 'SECRET';
+
+
+/**
+ * middleware for token validation
+ * @returns {void}
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Function} next
+ */
+export default function authenticationMiddleware(req, res, next) {
+  const token = req.headers.token;
+  if (!token) {
+    res.status(403).send('Not allowed');
+  }
+  jwt.verify(token, jwtSecret, (error) => {
```

Comment:
I believe this callback accepts (error, decoded).

you need to check decoded.expires to know if the token has expired
if it has not expired, you need to set req.decoded to decoded, so it is available in the next controller